microcosm/registry.py

```diff
@@ -19,6 +18,12 @@
 from microcosm.typing import Component
 
 
+try:
+    from importlib_metadata import entry_points as iter_entry_points  # type: ignore
+except ImportError:
+    from importlib.metadata import entry_points as iter_entry_points  # type: ignore
```

Comment:
how come doing the same thing again would work? at least add some comment to explain?